### PR TITLE
Fix redirect issue on accept and cancel rate plan actions.

### PR DESCRIPTION
**Pull request by minnur**

### CODE REVIEW

File: src/Entity/Subscription.php
Comment (Jaesin) [inline src/Entity/Subscription.php]: This looks very familiar. =)  See: https://github.com/apigee/apigee-m10n-drupal/pull/85/commits/24f17c300bf946db38a1c2cd09a4c7bb73411a1e

You could cherry pick `24f17c3` but it might be best to wait until the PR for 555 is merged and then use `$this->entity->getOwnerId()` where the UID is needed. Only because implmenting `EntityOwnerInterface` with the current handlers might have some unexpected effects.

**Comment by minnur** [comment]
I am going to re-implement this when https://github.com/apigee/apigee-m10n-drupal/pull/85 merged

**Review by Jaesin — APPROVED** [review]
Rebased to fix merge conflict. Manually tested.

**Pull request merged**
